include/internal/dso.h

```diff
@@ -163,71 +164,8 @@ DSO *DSO_dsobyaddr(void *addr, int flags);
  */
 void *DSO_global_lookup(const char *name);
 
-/* BEGIN ERROR CODES */
-/*
- * The following lines are auto generated by the script mkerr.pl. Any changes
- * made after this point may be overwritten when the script is next run.
- */
-
 int ERR_load_DSO_strings(void);
```

Comment:
When I look at `engine.h`, I find that the inclusion of `engineerr.h` is at *exactly* the same guarding level as the declaration of `ERR_load_ENGINE_strings()`...

But if what you say would be true, it would mean that `ERR_load_ENGINE_strings()` would be declared twice with different guardings, which I find somewhat questionable.